hooks/900-cleanup-etc-usr-var.chroot

```diff
@@ -47,6 +47,13 @@ rm -f /etc/mke2fs.conf
 # python deb install leftovers
 rm -rf /usr/local/lib/python*
 
+# we don't want any manpages or documentation
+rm -rf /usr/share/man /usr/share/groff /usr/share/info \
```

Comment:
Its interesting that most of these are pretty empty already (and some like linda do not exit on core18). I wonder if the base tarball we download already cleans the man-pages?

Reply:
Probably, might be something to check in free time. Should I remove those that are not existent from the list? It's a copy-paste from core and not sure if there's anything wrong in having those here just-in-case.